### PR TITLE
Adds licking

### PR DESCRIPTION
**Pull request by Superlagg**
## About The Pull Request

You can pull your tongue out with *lick, and use it to taste things.

Every atom can have a flavor, defined as an associated list, "flavor" = weight. 

Certain tongues are able to lick open wounds and apply a really awful bandage, and also heal a tiny bit of damage.

Licking batteries will zap you.

Licking light fixtures will burn you if intact, and fucking murder you if they're broken. I tried to make torches lickable, those decoration ones, but they're somehow fucked to the point that nothing can click on them properly. Oh well.

Also adds a slap ogg for some reason. Might use it later, idk.

### CODE REVIEW

File: code/game/objects/hand_items.dm [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+/// Course our first hand item would be a tongue
+/obj/item/hand_item/licker
+	name = "tongue"
+	desc = "Mlem."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "tonguenormal"
+	attack_verb = list("licked", "lapped", "mlemmed")
+	pokesound = 'sound/effects/gib_step.ogg'
+	siemens_coefficient = 5 // hewwo mistow ewectwic fence mlem mlem
```
[the remark this answers is not in the document]
Comment (Superlagg) [reply code/game/objects/hand_items.dm:27]: Good.

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -487,6 +490,31 @@
 	cost = 250
 	merge_type = /obj/item/stack/medical/gauze/cyborg
 
+/// ...
+/obj/item/stack/medical/gauze/lick
```
[the remark this answers is not in the document]
Comment (Superlagg) [reply code/game/objects/items/stacks/medical.dm:494]: I'll tie it to a quirk.

[the remark this answers is not in the document]
Comment (Superlagg) [reply code/game/objects/items/stacks/medical.dm:494]: (Also furry > human as it is~)

**Comment by Superlagg** [comment]
Okay so I made each function, healing and bandaging, their own -2 quirk.

Also lowered wound healing overall, cus with full decent coverings, you could heal 2.4ish wound per limb per second. Bit fast, should now be around 1.2ish, we'll see how that is.

Also turns out banages were fixing wounds twice as fast as they should cus of an oversight. They'll work properly now.

**Comment by bearrrrrrrr** [comment]
based

**Review by Tk420634 — APPROVED** [review]

**Pull request merged**